File: kotlinamesj/src/main/java/com/nagopy/android/kotlinames/property/nullable/KNullableDoublePropertyName.java
package com.nagopy.android.kotlinames.property.nullable;

import com.nagopy.android.kotlinames.KNullablePropertyName;
import com.nagopy.android.kotlinames.KSortablePropertyName;

public class KNullableDoublePropertyName implements KNullablePropertyName<Double>, KSortablePropertyName<Double> {

    private final String name;

    public KNullableDoublePropertyName(String name) {
        this.name = name;
    }

    @Override
    public String name() {
        return name;
    }
}

File: kotlinames/src/main/kotlin/com/nagopy/android/kotlinames/property/nullable/KNullableDoublePropertyName.kt
package com.nagopy.android.kotlinames.property.nullable

import com.nagopy.android.kotlinames.KNullablePropertyName
import com.nagopy.android.kotlinames.KSortablePropertyName

class KNullableDoublePropertyName(val name: String) : KNullablePropertyName<Double>, KSortablePropertyName<Double> {

    override fun name(): String = name

}
